rbx_binary/src/deserializer.rs

```diff
@@ -909,7 +783,7 @@ impl<R: Read> BinaryDeserializer<R> {
                                 ),
                             ));
                         } else {
-                            let special_case = special_case_to_rotation(id).ok_or_else(|| {
+                            let special_case = from_basic_rotation_id(id).ok_or_else(|| {
```

Comment:
This will error for `id > 0x24`, but not for something like `id == 0x04` or `id == 0x24`. Maybe a little funky?

Reply:
I don't think I like this behavior, so we're going back to the match.

Reply:
On the other hand, we could guard against these - but that's probably less clear than simply enumerating all the valid ids in the match expression.